Cds 92419 (#6700)
harness/developer-hub: docs/platform/git-experience/git-experience-overview.md

#### Git connector

:::note

> If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repo, you will not require a Git Connector and can skip this step!

Reviewer: ```suggestion
If you are using [Harness Code Repository](/docs/code-repository/) as your Git Repo, you won't need a Git Connector and can skip this step.
```
